static/variables/ui-variables.less

```diff
@@ -79,6 +79,19 @@
 
 @tab-height: 30px;
 
+// TitleBar
```

Comment:
Not sure about adding lots of new variables to `ui-variables.less`. Since this file gets loaded into many packages, we couldn't make any changes to it later. It's a bit like a public API. For now, or at least till we settle on the "combined" option, might be safer to just keep everything inside `title-bar.less`.

Another idea would be to use `@tab-bar-background-color;`. Since the tab-bar is just underneath, it should match most of the time? And maybe just flat without a gradient. Usually the core styles only style layout and sizing, but don't have much of an opinion on gradients, shadows etc. Maybe something like

```less
.title-bar {
  color: @text-color;
  background-color: @tab-bar-background-color;
  border-bottom: 1px solid @tab-bar-border-color;

  .is-blurred & {
    color: @text-color-subtle;
  }
}
```

Then themes that don't like it, can override it with their own styles.